src/JointAdapter.cpp

```diff
@@ -93,6 +92,10 @@ void JointVelocityAdapter::update(
 
   if (commandedVelocity > mUpperVelLimit || commandedVelocity < mLowerVelLimit)
   {
+    std::cout << "Desired Position: " << desiredPosition << std::endl;
```

Comment:
Is this intentional? Why not combine w/ the error string below?

Reply:
This is intentional (at least, the position printing is, I'll remove the velocity printing).

Technically, only the desired velocity causes the error. This is debugging information. I'll move it to a ROS_ERROR_STREAM so it outputs to stderr.